iocore/net/SSLUtils.cc

```diff
@@ -1758,7 +1760,7 @@ SSLInitServerContext(const SSLConfigParams *params, const ssl_user_config *sslMu
       } else if (strcmp(sslMultCertSettings->dialog, "builtin") == 0) {
         passwd_cb = ssl_private_key_passphrase_callback_builtin;
       } else { // unknown config
-        SSLError("unknown " SSL_KEY_DIALOG " configuration value '%s'", (const char *)sslMultCertSettings->dialog);
+        SSLError("unknown %s configuration value '%s'", SSL_KEY_DIALOG.data(), (const char *)sslMultCertSettings->dialog);
```

Comment:
This will work in this case because of how the view is constructed, but not in general.

Reply:
Should we instead to the string format that includes a length?

Reply:
Not worth it in this case, my goal was to make sure you knew you were "getting lucky". It's a thing I've abused elsewhere. Eventually, when it's all view based, this should be fixed, but we'll just have to live with a mixed state for now.